### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by stemangiola**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -5,8 +5,6 @@ output: github_document
 
 `CuratedAtlasQuery` is a query interface that allow the programmatic exploration and retrieval of the harmonised, curated and reannotated CELLxGENE single-cell human cell atlas. Data can be retrieved at cell, sample, or dataset levels based on filtering criteria. 
 
-# Query interface
```
Comment (multimeric) [inline README.Rmd:8]: Does this `README.Rmd` need to exist? It's just an artifact of knitting the vignette, no?

File: DESCRIPTION [hunk 2]
```diff
@@ -8,6 +8,56 @@ Authors@R: c(
...
+    person(
+        "Michael",
+        "Milton",
+        email = "milton.m@wehi.edu.au",
+        role = c("aut", "rev")
+    ),
+    person(
+        "Martin",
+        "Morgan",
+        email = "",
```
Comment (multimeric) [inline DESCRIPTION:21]: `email` is only needed for "`cre`" so you can probably omit this.

File: README.Rmd (hunk 1 above)
Comment (stemangiola) [reply README.Rmd:8]: I have splitted  README.Rmd (used for the README) and Introduction.Rmd for the veginette. As their image paths are different, and it is easier to manage now.

I usually include README.Rmd in the git repo.

Comment (multimeric) [reply README.Rmd:8]: Ah okay.

File: DESCRIPTION (hunk 2 above)
Comment (stemangiola) [reply DESCRIPTION]: I have added all emails. Should I eliminate all emails except mine. 

Would people prefer to have emails included or excluded?

File: README.Rmd (hunk 1 above)
Comment (multimeric) [reply README.Rmd:8]: I would be a bit wary about redundancy between these two files. e.g. if we change the API we will now have to change the examples in both of these `.Rmd` files.

File: DESCRIPTION (hunk 2 above)
Comment (multimeric) [reply DESCRIPTION]: I don't mind having mine. Probably Julie doesn't need to have hers.

File: README.Rmd (hunk 1 above)
Comment (stemangiola) [reply README.Rmd:8]: Yes this is a discussion I have being having for the last 3 years, and Bioconductor might have an elegant solution that I am not aware.

But I would say this redundancy is not that bad, and we can focus our energies elsewhere for the moment, leaving these details down the line.

**Pull request merged**